app/models/main.py

```diff
@@ -997,6 +998,7 @@ def serialize(self):
             'supplierName': self.supplier.name,
             'frameworkSlug': self.framework.slug,
             'frameworkFramework': self.framework.framework,
+            'frameworkFamily': self.framework.framework,
```

Comment:
I think we should add the `frameworkFamily` key to the `Framework.serialize()` method too. At the moment we just have `framework` in there - is the plan to eventually deprecate that field in favour of `framework_family` (as `self.framework.framework` is where all this confusion came from in the first place)?

Reply:
D'oh! Yep, that's a super important place to have it. 👍 

Will add.